snuba/datasets/discover.py

```diff
@@ -284,6 +285,7 @@ def get_query_processors(self) -> Sequence[QueryProcessor]:
 
         return [
             BasicFunctionsProcessor(),
+            ApdexProcessor(),
```

Comment:
What happens if we have an apdex query but events table gets selected? I guess it would give an error right now?

Reply:
Can it happen ? If we only support duration, shouldn't that always select transactions ?

Reply:
Unfortunately it could happen since: 1) it's possible to build a query that requests a transaction only column on the events dataset explicitly in Discover 2, or 2) The wrong dataset is picked because this syntax is not recognized as a column in the current syntax.